marimo/_plugins/stateless/audio.py

```diff
@@ -49,6 +102,13 @@ def audio(
             resolved_src = mo_data.audio(
                 f.read(), ext=os.path.splitext(src)[1]
             ).url
+    elif isinstance(src, np.ndarray):
```

Comment:
we should first check if numpy is imported with `DependencyManager.numpy.imported()`. 

i guess we don't have any existing tests. i can follow-up to create some tests for this file afterwards (unless you'd like to add them)